2021/45xxx/CVE-2021-45730.json

```diff
@@ -1,18 +1,81 @@
 {
```

Comment:
MISC or CONFIRM, but name != url for reference.

Reply:
I didn’t put URL as the name, can you please elaborate?